src/completion.rs

```diff
@@ -294,12 +297,19 @@ fn filename_complete(
     let dir_path = Path::new(dir_name);
     let dir = if dir_path.starts_with("~") {
         // ~[/...]
-        if let Some(home) = home_dir() {
-            match dir_path.strip_prefix("~") {
-                Ok(rel_path) => home.join(rel_path),
-                _ => home,
+        #[cfg(feature = "with-dirs")]
+        {
+            if let Some(home) = home_dir() {
+                match dir_path.strip_prefix("~") {
+                    Ok(rel_path) => home.join(rel_path),
+                    _ => home,
+                }
+            } else {
+                dir_path.to_path_buf()
             }
-        } else {
+        }
+        #[cfg(not(feature = "dir"))]
```

Comment:
`#[cfg(not(feature = "with-dirs"))]`

Reply:
got bit by last minute find and replaces... sorry about that.